notebooks/rxrx3_core_benchmarks_openphenom.ipynb

```diff
@@ -0,0 +1,2858 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #3.    openphenom_embeddings = pd.read_parquet("data/OpenPhenom_rxrx3-core_embeddings.parquet")  # download from huggingface
</pre><p>Do we want to add an example of how to download the data from hugging face? Or a link to the hf repo?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/recursionpharma/EFAAR_benchmarking/pull/84/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>